layers/meta-balena-tm3/recipes-kernel/linux/linux-tm3_%.bbappend

```diff
@@ -1,6 +1,33 @@
 inherit kernel-balena
 
 BALENA_CONFIGS[rtl8192cu] = ""
+BALENA_CONFIGS[ralink] = ""
 BALENA_CONFIGS_DEPS[ralink] = ""
-BALENA_CONFIGS_DEPS[brcmfmac] = ""
 BALENA_CONFIGS[brcmfmac] = ""
+BALENA_CONFIGS_DEPS[brcmfmac] = ""
+
+# Disabling nfsfs, as got following error, with it enabled -
+# ERROR: "fivm_open_verify" [fs/nfsd/nfsd.ko] undefined!
+BALENA_CONFIGS[nfsfs] = ""
+
+# Disabling uprobes, as found CONFIG_UPROBE_EVENTS
+# didn't appear to exist for our kernel
+BALENA_CONFIGS[uprobes] = ""
+
+# Disabling nf_tables, as found some config options
+# such as CONFIG_NFT_TUNNEL don't appear to exist for
+# our kernel
+BALENA_CONFIGS[nf_tables] = ""
```

Comment:
We should not disable this one, but rather check to see what is the underlying issue. Is CONFIG_NFT_TUNNEL the only one that cannot be enabled?